.drone.star

```diff
@@ -133,6 +133,63 @@ config = {
 				},
 			]
 		},
+		'core-api-acceptance': {
+			'suites': [
+				'apiAll',
```

Comment:
where does this come form?
How it works? (JFI)

Reply:
Seems like a magic word that just brings in all api tests from Core. Maybe @phil-davis  knows more about it.

Reply:
when you specify `'runCoreTests': True` then the starlark creates `numberOfParts` pipelines and names each pipeline like `apiAll-35-1-master-7.2`

`run.sh` in core knows how to select the suites from all the `api*` suites in core. If needed it will run multiple suites in a single pipeline...